Tests/SwiftDocCTests/Test Bundles/TestBundle.docc/Test-snippets.symbols.json

```diff
@@ -1 +1 @@
-{"metadata":{"generator":"SwiftPM","formatVersion":{"major":0,"minor":1,"patch":0}},"relationships":[{"kind":"memberOf","source":"$snippet__Test.FirstGroup.MySnippet","target":"$snippet__Test.FirstGroup","targetFallback":null}],"symbols":[{"accessLevel":"public","kind":{"displayName":"Snippet Group","identifier":"snippetGroup"},"pathComponents":["FirstGroup"],"identifier":{"precise":"$snippet__Test.FirstGroup","interfaceLanguage":"swift"},"names":{"title":"FirstGroup"},"docComment":{"lines":[{"text":""}]}},{"docComment":{"lines":[{"text":"Does a foo. "}]},"snippet":{"chunks":[{"code":"func foo() {\n  print(\"Hello, world!\")\n}","language":"swift"}]},"identifier":{"precise":"$snippet__Test.FirstGroup.MySnippet","interfaceLanguage":"swift"},"kind":{"displayName":"Snippet","identifier":"snippet"},"accessLevel":"public","pathComponents":["Test","FirstGroup","MySnippet"],"names":{"title":"MySnippet"}}],"module":{"name":"Test","platform":{}}}
```

Comment:
The extra module name in the path components here is incorrect. Notice how the other symbol doesn't have it.